integrations/serverless-integrating-service-mesh/serverless-integrating-ossm-3-x-setup.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="serverless-integrating-ossm-3-x-setup"]
+= Integrating Service Mesh 3.x with OpenShift Serverless
+include::_attributes/common-attributes.adoc[]
+:context: serverless-integrating-ossm-3-x-setup
+
+toc::[]
+
+[role="_abstract"]
+The {ServerlessOperatorName} provides Kourier as the default ingress for Knative. However, you can use {SMProductShortName} with {ServerlessProductName} whether Kourier is enabled or not. Integrating with Kourier disabled allows you to configure additional networking and routing options that the Kourier ingress does not support, such as mTLS functionality.
+
+Note the following assumptions and limitations:
+
+* All Knative internal components, as well as Knative Services, are part of the {SMProductShortName} and have sidecars injection enabled. This means that strict mTLS is enforced within the whole mesh. All requests to Knative Services require an mTLS connection, with the client having to send its certificate, except calls coming from OpenShift Routing.
+
+* {ServerlessProductName} with {SMProductShortName} integration can only target *one* service mesh. Multiple meshes can be present in the cluster, but {ServerlessProductName} is only available on one of them.
```

Comment:
```suggestion
* {ServerlessProductName} with {SMProductShortName} integration can only target one service mesh. Multiple meshes can be present in the cluster, but {ServerlessProductName} is only available on one of them.
```
No need to highlight *one* here unless it is a UI element. Leaving it upto you to check and decide. :)